.osx

```diff
@@ -326,6 +326,9 @@ sudo nvram boot-args="mbasd=1"
 # Show the ~/Library folder
 chflags nohidden ~/Library
 
+# Show the /Volumes folder
+chflags nohidden /Volumes
```

Comment:
Thanks for pointing it out. I wrongly assumed that asking for the administrator password in the very beginning is enough